dist/stat_data.py

```diff
@@ -335,6 +339,41 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Checkpoint statistics
```

Comment:
Is there a reason why the checkpoint-related statistics are split up into two different sections? Or was this how it was structured previously as well?

Reply:
Are you referring to the checkpoint/checkpoint cleanup split? If so, that was always split, and there were enough stats in the cleanup category that it didn't seem worth combining them. @sueloverso has also suggested this, so I'll combine them and see how it looks.

Reply:
No, I meant that there are two groups of just checkpoint statistics (not related to checkpoint cleanup). One of them is inside `conn_stats`, and the other is in `conn_dsrc_stats`.

Reply:
Oh, right - those two are per-dhandle, so they can't be in the same section as the connection-wide stats.